packages/babel-parser/src/plugins/typescript/errors.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+export default {
+  TS_ERR_0001: {
+    code: "TS_ERR_0001",
```

Comment:
I would prefer this to be a `Symbol()`, to prevent people from relying on them. We should discuss if we want to give every error a public code, or to keep it internal.